downstream/modules/platform/proc-controller-building-nodes-scenarios.adoc

```diff
@@ -0,0 +1,49 @@
+[id="controller-building-nodes-scenarios"]
+
+= Building nodes scenarios
+
+Learn how to manage nodes in the following scenarios.
+
+.Procedure
+
+* Click the (image:plus_icon_dark.png[Plus icon,15,15]) icon on the parent node to add a sibling node:
+
+image::ug-wf-create-sibling-node.png[Create sibling node]
+
+* Hover over the line that connects two nodes and click the plus (image:plus_icon_dark.png[Plus icon,15,15]), to insert another node in between nodes.
+Clicking the plus (image:plus_icon_dark.png[Plus icon,15,15]) icon automatically inserts the node between the two nodes:
+
+image::ug-wf-editor-insert-node-template.png[Insert node template]
+
+* Click btn:[START] again, to add a root node to depict a split scenario:
+
+image::ug-wf-create-new-add-template-split.png[Node split scenario]
+
+* At any node where you want to create a split scenario, hover over the node from which the split scenario begins and click the plus (image:plus_icon_dark.png[Plus icon,15,15]) icon.
+This adds adds multiple nodes from the same parent node, creating sibling nodes:
+
+image::ug-wf-create-siblings.png[Node create siblings]
+
+[NOTE]
+====
+When adding a new node, the btn:[PROMPT] option applies to workflow templates as well. 
+Workflow templates will prompt for inventory and surveys.
+====
+
+* You can undo the last inserted node by using one of these methods:
+. Click on another node without making a selection. 
+. Click btn:[Cancel].
+
+The following is an example of a workflow that contains all three types of jobs initiated by a job template that if it fails to run, you must proceed to the project sync job.
```

Comment:
This is a bit of a confusing sentence.

The following example workflow contains all three types of jobs initiated by a job template. If it fails to run, you must protect the sync job.